source/seismology/introduction.rst

```diff
@@ -1,2 +1,88 @@
 简介
 ====
+
+:本节贡献者: |姚家园|\（作者）、
+             |田冬冬|\（作者）
+:最近更新日期: 2021-04-23
+:预计阅读时间: 10 分钟
+
+----
+
+地震学是什么？
+--------------
+
+Keiiti Aki 和 Paul G. Richards 编写的经典地震学教材
+《Quantitative Seismology: Theory and Methods》对地震学的定义如下：
+
+  **Seismology is a science based on data called seismograms,
+  which are records of mechanical vibrations of the Earth.**
+
+  地震学是一门基于记录着地球机械振动的地震图的科学。
+
+  **These vibrations may be caused artificially by man-made explosions,
+  or they may be caused naturally by earthquakes and volcanic eruptions.**
+
+  振动可能是人工爆炸造成的，也可能是地震和火山喷发造成的。
+
+可以看出，地震学的研究工具是地震图，即地震仪记录的地球机械振动，而产生这些振动的源
+（一般称为\ **震源**\ ）可能是人工爆炸、地震和火山喷发等。震源激发的地球机械振动
+（一般称为\ **地震波**\ ）传播到地球内部，并最终被布设在地表或地下的地震仪
+（一般称为\ **地震台站**\ 或简称\ **台站**\ ） 记录下来。
+
+1994 年 1 月 17 日，美国加利福尼亚州 Northridge 附近发生了 `6.7 级地震 <https://earthquake.usgs.gov/earthquakes/eventpage/ci3144585/>`__，
+其释放出的能量相当于将近 20 亿千克高爆炸药。该地震产生的地震波经过地球内部传播后，
+被世界各地的地震台站记录到。一些地震波的传播路径以及全球台站记录的地表振动如下图所示。
+
+.. figure:: seismic-waves.jpg
+   :alt: 地震激发的地震波路径和波形
+   :width: 95%
+   :align: center
+
+   引自 `Exploring the Earth Using Seismology <https://www.iris.edu/hq/inclass/fact-sheet/exploring_earth_using_seismology>`__
```

Comment:
The figure has a low resolution. I would recommend converting the original PDF to an image and compress it.

Reply:
How about this resolution: https://migg-ntu.github.io/SeisTomo_Tutorials/_images/global-waves.png

Reply:
Better, but the image size is too large.

Reply:
done in 0f13d63. 179 kb.